tt_metal/impl/dispatch/kernels/cq_prefetch.cpp

```diff
@@ -903,6 +925,84 @@ static uint32_t process_relay_inline_exec_buf_cmd(uint32_t& cmd_ptr,
     return stride;
 }
 
+// This version of inline sends inline data to the dispatcher but doesn't flush the page to the dispatcher
+// This is used to assemble dispatcher commands when data comes out of band, eg, reading from DRAM
+// That means this command is stateful, incorrect use will be...bad
+// NOTE: this routine assumes we're sending a command header and that is LESS THAN A PAGE
+static uint32_t process_relay_inline_noflush_exec_buf_cmd(uint32_t& cmd_ptr,
+                                                 uint32_t& dispatch_data_ptr) {
```

Comment:
I think I should have named these "process_exec_buf_etc_cmd", care to fix if you agree?
I guess this has its own implementation because the size is known so the looping is simpler (less overhead), right?
can you add a comment about why this routine exists (how the command can get split by reading from dram)